src/main/java/svc/controllers/GlobalExceptionHandler.java

```diff
@@ -27,7 +27,7 @@ public String handleNotFoundException(NotFoundException e){
 	@ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
 	@ExceptionHandler(Exception.class)
 	public String handleGeneralExceptions(Exception e){
-		return ("500: General Unhandled Exception");
+		return ("500: General Unhandled Exception<br>"+e.getMessage());
```

Comment:
Yes, this was helping me during initial debugging.  What is the practice for logging or do we have a strategy in place already for the code?